core/src/main/res/values/strings.xml

```diff
@@ -37,6 +37,9 @@
     <string name="statistics_mode_normal">Calculate duration that was actually played. Playing twice is counted twice, while marking as played is not counted</string>
     <string name="statistics_mode_count_all">Sum up all podcasts marked as played</string>
     <string name="statistics_speed_not_counted">Notice: Playback speed is never taken into account.</string>
+    <string name="statistics_reset_data">Reset statistics data</string>
+    <string name="statistics_reset_data_title">Reset statistics data</string>
```

Comment:
Can you please just use one of the strings? Having two identical strings could lead to inconsistencies and will definitely lead to more work for the translators.

Reply:
Done